packages/hardhat-viem/test/chains.ts

```diff
@@ -52,41 +52,41 @@ describe("chains", () => {
       expect(chain).to.deep.equal(chains.foundry);
     });
 
-    it("should throw if the chain id is 31337 and the network is neither hardhat nor foundry", async () => {
```

Comment:
This diff is hard to read. For reference, I changed the last test from `"should throw if the chain id is not 31337 and there are multiple chains with that id"` to `"should return the first matching chain if the chain id is not 31337 and there are multiple chains with that id"` and moved it to the top to keep the error tests grouped at the bottom.